src/table/utils/handle-key-press.ts

```diff
@@ -128,6 +129,12 @@ export const handleHeadKeyDown = ({
       // Space bar / Enter: update the sorting
       changeSortOrder(layout, column);
       break;
+    case KeyCodes.C: {
+      areBasicFeaturesEnabled &&
+        (evt.ctrlKey || evt.metaKey) &&
+        copyCellValue((evt.target as HTMLElement).children[0].firstChild?.textContent as string);
```

Comment:
FirstChild is optional here but still the best option we have to get the textContent for Head cells without being disturbed by SortLabel span content.